httpdirfs: 1.2.5 -> 1.2.6

anthonyroussel opened the pull request:
https://github.com/fangfufu/httpdirfs/compare/1.2.5...1.2.6

Note for reviewers: MacOS support has been removed upstream with 1.2.6 release.


<!--
^ Please summarise the changes you have done and explain why they are necessary here ^

For package updates please link to a changelog or describe changes, this helps your fellow maintainers discover breaking updates.
For new packages please briefly describe the package or provide a link to its homepage.
-->

## Things done

<!-- Please check what applies. Note that these are not hard requirements but merely serve as information for reviewers. -->

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- For non-Linux: Is sandboxing enabled in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
  - [ ] `sandbox = relaxed`
  - [ ] `sandbox = true`
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#linking-nixos-module-tests-to-a-package) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [24.11 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2411.section.md) (or backporting [23.11](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2311.section.md) and [24.05](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2405.section.md) Release notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

<!--
To help with the large amounts of pull requests, we would appreciate your
reviews of other pull requests, especially simple package updates. Just leave a
comment describing what you have tested in the relevant package/service.
Reviewing helps to reduce the average time-to-merge for everyone.
Thanks a lot if you do!

List of open PRs: https://github.com/NixOS/nixpkgs/pulls
Reviewing guidelines: https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#reviewing-contributions
-->

---

Add a :+1: [reaction] to [pull requests you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[pull requests you find important]: https://github.com/NixOS/nixpkgs/pulls?q=is%3Aopen+sort%3Areactions-%2B1-desc

Artturin commented:
> Note for reviewers: MacOS support has been removed upstream with 1.2.6 release.

Intentionally or accidentally? Linking a issue/pr/commit in the commit msg would be good

anthonyroussel commented:
> Intentionally or accidentally? Linking a issue/pr/commit in the commit msg would be good

@Artturin Support for other operating systems has been intentionally removed upstream:

> I don't have the resources to test out compilation for Linux distributions
other than Debian. I also do not have the resources to test out compilation for
FreeBSD or macOS. Thereforce I have removed the instruction on how to compile
for these operating systems in the README for now. Please feel free to send me a
pull request to add them back in.

See https://github.com/fangfufu/httpdirfs/compare/1.2.5...1.2.6 and https://github.com/fangfufu/httpdirfs?tab=readme-ov-file#other-operating-systems

I'll add this to the commit message.

anthonyroussel committed:
httpdirfs: 1.2.5 -> 1.2.6

Changelog: https://github.com/fangfufu/httpdirfs/compare/1.2.5...1.2.6

Unfortunately, MacOS support has been removed intentionally upstream with this release:

> I don't have the resources to test out compilation for Linux distributions
> other than Debian. I also do not have the resources to test out compilation for
> FreeBSD or macOS. Thereforce I have removed the instruction on how to compile
> for these operating systems in the README for now. Please feel free to send me a
> pull request to add them back in.

See https://github.com/fangfufu/httpdirfs?tab=readme-ov-file#other-operating-systems
